packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -658,13 +656,13 @@ export enum SimulatorDeviceSet {
   RN_IDE,
 }
 
-export async function createSimulator(
-  modelId: string,
+export async function createSimulatorWithRuntimeId(
```

Comment:
VERY NIT:
I don't see the point of making this a separate exported function. Can't the caller just call `createSimulator(...).UDID` if they want just the `UDID`?
Right now you have two similarly called functions where it's not clear what's the difference between them without reading the implementation.

Reply:
one method takes `runtimeId: string` as argument whereas the other takes an object. The problem here is that when we create temp simulator, we only care about the UDID and so even if we wanted to only extract that we'd still need to have an instance of runtime. In order to get instance of runtime we'd have to pass it down to the build likely via build config and this would impact caching etc. We could also create runtime in place but that'd require an extra simctl call to retrieve the runtime metadata

Reply:
I see, I missed this. I thought the parameters for both were the same.
It's fine then.